climada/hazard/tc_tracks_forecast.py

```diff
@@ -83,7 +87,8 @@ class TCForecast(TCTracks):
         data (list(xarray.Dataset)): Same as in parent class, adding the
             following attributes
                 - ensemble_member (int)
-                - is_ensemble (bool)
+                - is_ensemble (bool; if True, the simulation is a high resolution
+                               deterministic run)
```

Comment:
(if I've understood this correctly)
```suggestion
                - is_ensemble (bool; if False, the simulation is a high resolution
                               deterministic run)
```